R/SummaryForestPlotsTab.R

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
The plots are quite small when viewed side by side on a laptop screen - I would like to get some other people's thoughts on having the plots side-by-side or top-to-bottom.

Reply:
That's a good point about the size. Everything else in the app is side-by side, so that becomes part of the user's expectations using the app. I don't have a strong suggestion on how to lay them out better without breaking that user experience.

Reply:
Some of the tabs are top-to-bottom i.e. tab 2b which is not that dissimilar to tab 2d.

Reply:
I think we need to discuss making plots reactive (CSS) so they grow an shrink with the window. Another idea I think would be good is to have an on click event on the plots that shows that plot as a modal (in window pop up), and adding an on:hover CSS to change the curser to a magnifying glass.

Reply:
I don't disagree @RyanJField , but I think that should be a separate issue (to be implemented across the app).
Having thought about the layout more, I would argue that the summary forest plots should be laid out top-to-bottom because the width of the plot increases with increasing numbers of treatments.

Reply:
I've made them atop one another, rather than side-by-side. I like the idea of the better plot interactions, but I agree with Naomi that it would be best as a separate task (Which I think is what you meant).